src/components/Descope.tsx

```diff
@@ -93,6 +94,12 @@ const Descope = React.forwardRef<HTMLElement, DescopeProps>(
 			};
 		}, [innerRef, onError, handleSuccess]);
 
+		useEffect(() => {
```

Comment:
How about adding tests ? Should be simple I think - just make sure that the wc instance has this property